specs/latest/1.0/index.html

```diff
@@ -723,14 +723,19 @@ <h2 class="no-toc">Table of contents</h2>
         The <code>WebGLContextAttributes</code> dictionary contains drawing surface attributes and
         is passed as the second parameter to <code>getContext</code>.
     </p>
-    <pre class="idl">dictionary <dfn id="WebGLContextAttributes">WebGLContextAttributes</dfn> {
+    <!-- The capitalization rules here match those of the crossOrigin
+    attribute, which was added to HTMLMediaElement on behalf of
+    WebGL. -->
+    <pre class="idl">enum PowerPreference { "default", "low-power", "high-performance" };
```

Comment:
I'm not sure if "balanced" describes what we'll be doing. On an ACPI system, "balanced" usually refers to automatic frequency scaling. If this mostly controls which GPU is used, "default" makes more sense.